src/libstd/sys/unix/c.rs

```diff
@@ -15,11 +15,26 @@
 
 pub use self::select::fd_set;
 pub use self::signal::{sigaction, siginfo, sigset_t};
+#[cfg(not(target_os = "nacl"))]
 pub use self::signal::{SA_ONSTACK, SA_RESTART, SA_RESETHAND, SA_NOCLDSTOP};
```

Comment:
Now that globs are stable, could this actually switch to using a `pub use self::signal::*` glob? That should avoid this #[cfg] stuff